WasatchNET/SPISpectrometer.cs

```diff
@@ -319,6 +319,9 @@ override internal bool open()
 
             pixels = (ushort)Unpack.toShort(pixelBytes);
 
+            if (pixels > 10000)
```

Comment:
Is this a check against 0xffff etc?

Reply:
Yeah, it's just a sanity check generally. The other one, where we explicitly look for an ACK of 0, is probably the more important one.